digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -965,12 +965,12 @@ object RoadAddressDAO {
         case None => ""
       })
       addressPS.setString(11, createdBy.getOrElse(address.modifiedBy.getOrElse("-")))
-      val (p1, p2) = (address.geometry.head, address.geometry.last)
-      addressPS.setDouble(12, p1.x)
-      addressPS.setDouble(13, p1.y)
+      val (p1, p2) = (address.geometry.headOption, address.geometry.lastOption)
```

Comment:
Option seems useless as we call .get after this every time.

Reply:
ah, yea forgot to remove those as well, had logic to add null but DB wouldn't accept it